setup.py

```diff
@@ -25,7 +25,7 @@
     ],
     dependency_links=[
         "https://github.com/Lyrositor/CCScriptWriter/tarball/master#egg=CCScriptWriter-1.2",
-        "https://github.com/tripped/ccscript_legacy/tarball/master#egg=ccscript-1.338"
+        "https://github.com/jamsilva/ccscript_legacy/tarball/master#egg=ccscript-1.338"
```

Comment:
Have you seen this PR? Perhaps you might consider merging it into your branch of ccscript_legacy? It has been a thorn in the sides of many fan translators who use Windows with non-English languages.

https://github.com/tripped/ccscript_legacy/pull/7/files

Reply:
Merged.